cmd/kube-apiserver/app/options/validation.go

```diff
@@ -36,6 +36,7 @@ import (
 // validateClusterIPFlags is expected to be called after Complete()
 func validateClusterIPFlags(options *ServerRunOptions) []error {
 	var errs []error
+	maxCIDRbits := 20
```

Comment:
should we add a comment explaining it?

Reply:
Here is a small description about why: https://github.com/kubernetes/kubernetes/pull/90391/files#diff-6696b3d19b862aab51fa1646fc7811abL78-L80 but I can add a short comment why we usee this variable.

Reply:
I added a comment, do you know when this limitation was introduced? I also couldn't find any document that references this limitation (maybe a gap in the docs?).

Reply:
honestly, I think that's there from the beginning to limit the size of the bitmaps in etcd, I've tried to remove the limitation using roaring bitmaps https://github.com/kubernetes/enhancements/pull/1534
 but we concluded it wasn't worth the effort because there isn't a real demand

Reply:
just one nit more, sorry, I think this should be a constant, but I defer to you the choice